src/main/java/com/mraof/minestuck/item/weapon/OnHitEffect.java

```diff
@@ -167,6 +167,39 @@ static OnHitEffect setOnFire(int duration)
 		return (itemStack, target, attacker) -> target.setFire(duration);
 	}
 	
+	static OnHitEffect armorBypassingDamageMod(float additionalDamage, EnumAspect aspect)
+	{
+		return (stack, target, attacker) -> {
+			DamageSource source;
+			float damage = additionalDamage * 3.3F;
+			
+			if(attacker instanceof PlayerEntity)
```

Comment:
Maybe change this to a `ServerPlayerEntity` since that's what you're using in the cast below. (and maybe add a `ServerPlayerEntity player = (ServerPlayerEntity) attacker;` so that you don't have to do multiple casts.